src/test/java/uk/gov/hmcts/payment/processors/DeadLetterQueueProcessorTest.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.payment.processors;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DeadLetterQueueProcessorTest extends BaseIntegrationTest{
+
+    @Test
+    void happyPathSucceeds() {
+        stubForS2s();
+        stubForCardStatusUpdate();
+
+        JobProcessorConfiguration configuration = new MockJobProcessorConfiguration(
+                "http://localhost:" + s2sWiremock.port(),
```

Comment:
This is an Integration Test Wiremocking should not be done in an Unit Test...

Reply:
The relevant piece of code is just a re-direction to a specific URL, hence an integration test is done here.

Reply:
Integration Test have a specific module to be placed in as part of the HMCTS Spring Component Template and the cant be placed in the Unit Test location.